README.md

```diff
@@ -1,54 +1,76 @@
+    - [ ] Dependencies on other packages
+      - [x] tf2_kdl https://github.com/ros2/geometry2/pull/90
+      - [x] eigen_stl_containers https://github.com/AcutronicRobotics/eigen_stl_containers/tree/ros2
+      - [x] geometric_shapes https://github.com/ros-planning/geometric_shapes/pull/96
+      - [x] random_numbers https://github.com/ros-planning/random_numbers/pull/12
+      - [x] srdfdom (contributed by @anasarrak, @vmayoral and @ahcorde) https://github.com/ros-planning/srdfdom/pull/45
+      - [x] urdf_parser_py https://github.com/ros/urdf_parser_py/pull/41
+      - [x] Created a ROS 2 version (with package.xml) of urdfdom_headers https://github.com/AcutronicRobotics/urdfdom_headers/tree/ros2
+      - [x] octomap https://github.com/AcutronicRobotics/octomap
+        - [x]  octomap
+        - [ ]  octovis
+        - [ ]  dynamicEDT3D
+  - [ ] Convert moveit_kinematics
+  - [ ] Convert moveit_planners_ompl
+  - [ ] Convert moveit_ros_planning
+  - [ ] Convert moveit_ros_planning_interface
+  - [ ] Convert moveit_ros_benchmarks
+  - [ ] Convert moveit_ros_control_interface
+  - [ ] Convert moveit_ros_manipulation
+  - [ ] Convert moveit_ros_move_group
+  - [ ] Convert moveit_ros_perception
+  - [ ] Convert moveit_ros_robot_interaction
+  - [ ] Convert moveit_ros_visualization
+  - [ ] Convert moveit_ros_warehouse
+  - [ ] Convert moveit_setup_assistant
+  - [ ] Convert moveit_simple_controller_manager
+  - [ ] Convert moveit_visual_tools
+  - [ ] Convert moveit_task_constructor
+  - [ ] Convert moveit_resources
+  - [ ] Convert moveit_commander
+  - [ ] Convert moveit_fake_controller_manager
+- [ ] New features in ROS 2.0
+  - [ ] Migrate plugin architecture to ROS2 nodelets
+- [ ] Documentation
+  - [ ] Create tutorials
 
-service    | Indigo | Kinetic | Melodic
----------- | ------ | ------- | -------
-Travis     | [![Build Status](https://travis-ci.org/ros-planning/moveit.svg?branch=indigo-devel)](https://travis-ci.org/ros-planning/moveit/branches) | [![Build Status](https://travis-ci.org/ros-planning/moveit.svg?branch=kinetic-devel)](https://travis-ci.org/ros-planning/moveit/branches) | [![Build Status](https://travis-ci.org/ros-planning/moveit.svg?branch=melodic-devel)](https://travis-ci.org/ros-planning/moveit/branches) |
-build farm | [![Build Status](http://build.ros.org/buildStatus/icon?job=Idev__moveit__ubuntu_trusty_amd64)](http://build.ros.org/job/Idev__moveit__ubuntu_trusty_amd64) | [![Build Status](http://build.ros.org/buildStatus/icon?job=Kdev__moveit__ubuntu_xenial_amd64)](http://build.ros.org/job/Kdev__moveit__ubuntu_xenial_amd64) | [![Build Status](http://build.ros.org/buildStatus/icon?job=Mdev__moveit__ubuntu_bionic_amd64)](http://build.ros.org/job/Mdev__moveit__ubuntu_bionic_amd64) |
+## Continuous Integration Status
 
+[![Build Status](https://travis-ci.org/AcutronicRobotics/moveit2.svg?branch=ros2)](https://travis-ci.org/AcutronicRobotics/moveit2)
```

Comment:
I've just turned on Travis for moveit2:
https://travis-ci.org/ros-planning/moveit2

Reply:
https://github.com/ros-planning/moveit2/pull/1/commits/f925d83718b1533cebb9196829ca410399d415fc, please review. Note I pointed to `master` branch.